act/update_concept_dimension.sql

```diff
@@ -71,7 +71,7 @@ DELETE /*+  PARALLEL (20) */ from
 "&&metadata_schema"."HERON_TERMS"
 where C_FULLNAME = '\i2b2\Demographics\LESS_THAN_10\'
```

Comment:
Can you get rid of the files as you are building concpet_dimension in each SQL file? ?

Reply:
yes; I didn't quite finish that either, did I?